frame.go

```diff
@@ -2203,106 +2207,101 @@ func newCompressedFrame(uncompressedPayload []byte, isSelfContained bool, compre
 	if err != nil {
 		return nil, err
 	}
-	// skipping first 4 bytes because size of uncompressed payload now is written in frame header,
-	// not in the body of compressed envelope
+
+	/* Skip the first 4 bytes because the size of the uncompressed payload is written in the frame header, not in the
+	body of the compressed envelope
+	*/
```

Comment:
Let's follow the existing for this code base comment style. Just use // even for multi-line comments.